ember-power-calendar/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "ember-power-calendar",
+  "version": "0.21.0",
+  "description": "Powerful and customizable calendar component for Ember",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cibernox/ember-power-calendar.git"
+  },
+  "license": "MIT",
+  "author": "Miguel Camba",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": "./dist/*.js",
+    "./addon-main.js": "./addon-main.cjs",
+    "./_index.less": "./_index.less",
+    "./_index.scss": "./_index.scss",
+    "./less/base.less": "./less/base.less",
+    "./scss/base.scss": "./scss/base.scss",
+    "./vendor/*": "./vendor/*"
+  },
+  "files": [
+    "addon-main.cjs",
+    "declarations",
+    "dist",
+    "less",
+    "scss",
+    "vendor"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "prepack": "pnpm build",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepare": "pnpm build"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.8.7",
+    "decorator-transforms": "^1.0.1",
+    "@ember/render-modifiers": "^2.1.0",
+    "@embroider/macros": "^1.13.4",
+    "ember-assign-helper": "^0.4.0",
+    "ember-concurrency": "^3.1.0",
+    "ember-element-helper": "^0.8.5",
+    "ember-truth-helpers": "^4.0.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.7",
```

Comment:
I think you need to add the missing dependencies like ember-power-calendar-moment here to fix some of the errors

Reply:
no, this is correct so.. the addon doen't need to know which meta package is used. we can also remove the peerDependency because its not required

Reply:
ok interesting. thanks